src/app/pages/student/details/tabs/student-details-personal-info/student-details-personal-info.component.ts

```diff
@@ -81,24 +82,31 @@ export class StudentDetailsPersonalInfoComponent implements OnInit {
 
   async addStudent() {
     try {
+      this.saveFailed = false;
       await this.studentService.create(this.student);
-      this.router.navigate(['/student']);
+      this.changesWereSavedNotification();
     } catch (error) {
-      // TODO: implement error handling on UI
-      console.error('Error handling not implemented');
-      throw error;
+      this.saveFailed = true;
+      console.error('Error on add student', error);
     }
   }
 
   async updateStudent(student) {
     student.form.value._id = this.student._id;
     try {
+      this.saveFailed = false;
       await this.studentService.update(student.form.value);
-      this.router.navigate(['/student']);
+      this.changesWereSavedNotification();
     } catch (error) {
-      // TODO: implement error handling on UI
-      console.error('Error handling not implemented');
-      throw error;
+      this.saveFailed = true;
+      console.error('Error on update student', error);
     }
   }
+
+  changesWereSavedNotification() {
```

Comment:
I don't like solutions that based on timeout. Can you explain what this is doing? Can we have other code than timeout?

Reply:
@eran1205 
It changes the button to נשמר with green background only for 1000ms.
After the timout, the button will return to its original design.